From 4a38ac466db10eb192cbe04f6f33942e15065fa5 Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Thu, 17 Dec 2015 14:44:24 +0100
Subject: [PATCH 5/7] SCore: Add support for SPI boot

Add the support for an boot over an SPI-NOR-Flash to the DCD. With the
right fuses setted the processor will load the U-Boot from an
SPI-NOR-Flash into the RAM.

Ticket #871
---
 board/freescale/mx6sxscore/imximage.cfg | 14 +++-----------
 1 file changed, 3 insertions(+), 11 deletions(-)

diff --git a/board/freescale/mx6sxscore/imximage.cfg b/board/freescale/mx6sxscore/imximage.cfg
index 8f699ab..312203f 100644
--- a/board/freescale/mx6sxscore/imximage.cfg
+++ b/board/freescale/mx6sxscore/imximage.cfg
@@ -23,14 +23,7 @@ IMAGE_VERSION 2
  * spi/sd/nand/onenand, qspi/nor
  */
 
-// TODO: Add boot from Nand
-#ifdef CONFIG_SYS_BOOT_QSPI
-BOOT_FROM	qspi
-#elif defined(CONFIG_SYS_BOOT_NAND)
-BOOT_FROM   nand
-#else
-BOOT_FROM	sd
-#endif
+BOOT_FROM	spi
 
 #ifdef CONFIG_USE_PLUGIN
 /*PLUGIN    plugin-binary-file    IRAM_FREE_START_ADDR*/
@@ -56,13 +49,12 @@ CSF 0x2000
 /* Enable all clocks */
 DATA 4 0x020c4068 0xffffffff
 DATA 4 0x020c406c 0xffffffff
-//DATA 4 0x020c4070 0xffffffff
-DATA 4 0x020c4070 0xfc3fffff
+DATA 4 0x020c4070 0xffffffff
 DATA 4 0x020c4074 0xffffffff
 DATA 4 0x020c4078 0xffffffff
 DATA 4 0x020c407c 0xffffffff
 DATA 4 0x020c4080 0xffffffff
-//DATA 4 0x020c4084 0xffffffff
+DATA 4 0x020c4084 0xffffffff
 
 /* IOMUX */
 /* DDR IO TYPE */
-- 
1.9.1

